.github/workflows/deploy_production.yml

```
@@ -0,0 +1,123 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/SchemaStore/schemastore/master/src/schemas/json/github-workflow.json
+---
+name: Deploy CloudFormation stacks for production environment and development environment mirror
+
+on:
+    push:
+        branches:
+            - main
+
+    workflow_dispatch:
+
+concurrency:
+    group: ${{ github.ref }}
+    cancel-in-progress: true
+
+permissions:
+    id-token: write
+    contents: read
+    pull-requests: write
+    deployments: write
+    repository-projects: write
+
+env:
+    HUSKY: 0
+    NODE_ENV: production
+    HOSTED_ZONE_ID: ${{ secrets.HOSTED_ZONE_ID }}
+    CERTIFICATE_ARN: ${{ secrets.CERTIFICATE_ARN }}
+    MONGODB_URI_PROD: ${{ secrets.MONGODB_URI_PROD }}
+
+    # Turborepo credentials.
+    TURBO_API: ${{ vars.TURBO_API }}
+    TURBO_TOKEN: ${{ secrets.TURBO_TOKEN }}
+    TURBO_TEAM: team_antalmanac
+
+    # AWS credentials.
+    AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+    AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+    AWS_DEFAULT_REGION: us-east-1
```

Prod is deployed to us-west-1
```suggestion
    AWS_DEFAULT_REGION: us-west-1
```